run-build-functions.sh

```diff
@@ -166,8 +171,6 @@ run_npm_set_temp() {
 }
 
 run_npm() {
-  restore_cwd_cache node_modules "node modules"
```

Comment:
I am curious whether there was a reason this line was moved after `npm install -g npm`?
If not, we might want to keep the current place.

Reply:
There wasn't I'm not sure why I moved it honestly 😅 

Addressed in 68778bb9f90782fdfcdb4ac953bd6ff835bc8b30